fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/mv/InitMaterializationContextHook.java

```diff
@@ -70,7 +70,7 @@ private void initMaterializationContext(CascadesContext cascadesContext) {
         List<BaseTableInfo> usedBaseTables =
                 collectedTables.stream().map(BaseTableInfo::new).collect(Collectors.toList());
         Set<MTMV> availableMTMVs = Env.getCurrentEnv().getMtmvService().getRelationManager()
-                .getAvailableMTMVs(usedBaseTables);
+                .getAvailableMTMVs(usedBaseTables, cascadesContext.getConnectContext());
```

Comment:
should make sure that this connect connext should not be modified later

Reply:
ok